SU-HSDO/suhumsci
# SHS-5530: hb-stretch-vertical-linked-cards utility class should also work on private pages

# READY FOR REVIEW

## Summary
Make hb-stretch-vertical-linked-cards utility class work on private pages

## Need Review By (Date)
03/20

## Urgency
medium

## Steps to Test
1. Create a private page
2. Add a `Private Collection` component
3. Add 3 `Postcards` with the `Vertical Linked Card`
4. Remember to add the `hb-stretch-vertical-linked-cards` utility class
5. Confirm all the cards have the same height
6. Repeat the same for Public/Flexible Page to confirm everything still looks as expected there

## PR Checklist
- [PR Checklist](https://gist.github.com/sherakama/0ba17601381e3adbe0cad566ad4d80a5)
- [Humsci Basic PR Checklist](https://github.com/SU-HSDO/suhumsci/blob/develop/docs/HumsciBasicPRChecklist.md)
